cli: Коммандная оболочка

grannev opened the pull request:
Использование:
python3 -m src.utils.cli clean-html
По стандарту
Читает из data/raw/ml_ozon_counter_test.csv
Записывает в data/processed/ml_ozon_counter_test_processed.csv

Либо через тег --input-file "*.csv" --outpul-file "*.csv"

grannev committed:
cli: Коммандная оболочка
Использование:
python3 -m src.utils.cli clean-html
По стандарту
Читает из data/raw/ml_ozon_counter_test.csv
Записывает в data/processed/ml_ozon_counter_test_processed.csv

Либо через тег --input-file "*.csv" --outpul-file "*.csv"

senks69 commented:
Можно в pyproject.toml в [poetry.script], чтобы в окружении одной командой запускалось(без python3 cli.py)

grannev committed:
Добавлен скрипт poetry, исп-ие:
```Shell
poetry run cli clean
```
Дописаны методы clean-text и clean
Добавлен метод prepare-data для будущих обработок
